doc/howtos/software-update-server.rst

```diff
@@ -219,26 +217,24 @@ from the command in the Example is:
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/
 
   This folder contains both data from intermediate steps and the actual
-  SW Update stream.
-
-  Its location is::
+  SW Update stream located here::
 
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/www/
 
-  This folder will contain also data related to following builds and it is
-  the one that must be exposed to the device in need of maintenance, through
-  a web server, for example nginx or apache.
+  This folder will also contain data related to subsequent builds and
+  must be exposed through a web server (e.g., ``nginx`` or ``apache``) 
+  to the device running the SW update client for getting its maintentance.
```

Comment:
we need to say how the user will reach this server. Do we have some variable to define so swupd in the resulting image will find it? If not, then we must add it in the future (jira item) and meanwhile say that the user must use a specific swupd switch. If we do have a "swupd.timer" we need to mention that it must be updated as well.